MediaGalleryUi/composer.json

```diff
@@ -10,7 +10,6 @@
         "magento/module-media-gallery-ui-api": "*",
         "magento/module-media-gallery-api": "*",
         "magento/module-media-gallery-synchronization": "*",
```

Comment:
MediaGalleryUi should depend on MediaGallerySynchronizarionApi instead of MediaGallerySynchronizarion

Reply:
ok